Document/0x05b-Basic-Security_Testing.md

```diff
@@ -1297,22 +1297,56 @@ Check the [Termux Wiki](https://wiki.termux.com/wiki/Remote_Access "Termux Remot
+One of the easiest options is to download the APK from websites that mirror public applications from the Google Play Store. However, keep in mind that these sites are not official and there is no guarantee that the application hasn't been repackaged or contain malware. A few reputable websites that host APKs and are not known for modifying apps and even list SHA-1 and SHA-256 checksums of the apps are:
 
 - [APKMirror](https://apkmirror.com "APKMirror")
 - [APKPure](https://apkpure.com "APKPure")
 
 Beware that you do not have control over these sites and you cannot guarantee what they do in the future. Only use them if it's your only option left.
 
+##### Using gplaycli
+
+[gplaycli](https://github.com/matlink/gplaycli "gplaycli") is a Python based CLI tool to search, install and update Android applications from the Google Play Store. Follow the [installation steps](https://github.com/matlink/gplaycli#installation "gplaycli Installation") and you're ready to run it. gplaycli offers several options, please refer to its help (`-h`) for more information.
+
+If you're unsure about the package name (or AppID) of an app, you may perform a keyword based search for APKs (`-s`):
+
+```bash
+$ gplaycli -s "google keep"
+
+Title                          Creator     Size      Last Update  AppID                                    Version
+
+Google Keep - notes and lists  Google LLC  15.78MB   4 Sep 2019   com.google.android.keep                  193510330
+Maps - Navigate & Explore      Google LLC  35.25MB   16 May 2019  com.google.android.apps.maps             1016200134
+Google                         Google LLC  82.57MB   30 Aug 2019  com.google.android.googlequicksearchbox  301008048
+```
+
+Next, you can download (`-d`) the selected APK by specifying its AppID (add `-p` to show a progress bar and `-v` for verbosity):
+
+```bash
+$ gplaycli -p -v -d com.google.android.keep
+[INFO] GPlayCli version 3.26 [Python3.7.4]
+[INFO] Configuration file is ~/.config/gplaycli/gplaycli.conf
+[INFO] Device is bacon
+[INFO] Using cached token.
+[INFO] Using auto retrieved token to connect to API
+[INFO] 1 / 1 com.google.android.keep
+[################################] 15.78MB/15.78MB - 00:00:02 6.57MB/s/s
+[INFO] Download complete
+```
+
+The `com.google.android.keep.apk` file will be in your current directory. As you might imagine, this approach is a very convenient way to download APKs, especially with regards to automation.
+
+> You may use your own Google Play credentials or token. By default, gplaycli will use [an internally provided token](https://github.com/matlink/gplaycli/blob/3.26/gplaycli/gplaycli.py#L106 "gplaycli Fallback Token").  
+
```

Comment:
Please add a note that gplaycli does abide by the restrictions of your country: e.g. when an app is not available in the country you are running this, then gplaycli will not be able to show it in your queries. This is different from the website APKmirror for instance , where such rules are not enforced.

Reply:
added a note on regional restrictions.